doc/library/tensor/nnet/ctc.txt

```diff
@@ -0,0 +1,28 @@
+.. _libdoc_tensor_nnet_ctc:
+
+===============================================================================
+:mod:`ctc` -- Operation to compute CTC loss and its gradients
+===============================================================================
+
+.. note::
+
+    Usage of connectionist temporal classification (CTC) loss Op, requires that
+    the `warp-ctc <https://github.com/baidu-research/warp-ctc>`_ library is
+    available. Also, the `config.ctc.enabled` and `config.ctc.root` configuration
+    options must be appropriately set.
+
+.. note::
+
+    Unfortunately, Windows platforms are not yet supported by the underlying
+    library.
+
+.. module:: ctc
+   :platform: Unix
+   :synopsis: Connectionist temporal classification (CTC) loss Op, using the warp-ctc library
+.. moduleauthor:: `João Victor Risso <https://github.com/joaovictortr>`_
+
+.. autofunction:: theano.tensor.nnet.ctc.ctc
+.. autofunction:: theano.tensor.nnet.ctc.ConnectionistTemporalClassification
+
+.. automodule:: theano.tensor.nnet.ctc
```

Comment:
As you already organize doc in previous lines (with autofunction and autoclass), you don't need to use the automodule section here. It will just print the doc twice.